features/exploredash/src/main/res/values-es/strings-explore-dash.xml

```diff
@@ -34,7 +34,7 @@
     <string name="explore_option_buy">Comprar</string>
     <string name="explore_option_sell">Vender</string>
     <string name="explore_option_buy_sell">Comprar y Vender</string>
-    <string name="explore_menu_info">Información</string>
+    <string name="button_info">Información</string>
```

Comment:
In Transifex, what will happen is this:
1. we will push the source files, which will no longer have `explore_menu_info`, but instead `button_info`
2. `button_info` is new and no languages will have this string translated
3. Transifex will see that languages are missing `button_info`, but will see in memory how `"info"` was translated.  Then it will automatically fill out `button_info` for all languages that had a translation for `"info"`

We won't need to push the changes in this particular file to Transifex, but next time we pull translations, `button_info` should be included.